src/common/low_precision_transformations/tests/multiply_transformation.cpp

```diff
@@ -20,14 +20,49 @@
 #include "simple_low_precision_transformer.hpp"
 #include "lpt_ngraph_functions/multiply_function.hpp"
 
+#include "ngraph/pass/serialize.hpp"
+
 namespace {
 using namespace testing;
 using namespace ov;
 using namespace ov::pass;
 using namespace ngraph::builder::subgraph;
 
+class MultiplyBranch {
+public:
+    ngraph::builder::subgraph::Constant constant;
+    ngraph::element::Type input_precision;
+    ngraph::builder::subgraph::DequantizationOperations dequantization;
+};
```

Comment:
Please note that we don't cover the case with FQ on constant path by these tests. Should we do that?

Reply:
covered, thanks. FQ is supported